src/cowboy_clock.erl

```diff
@@ -83,10 +83,12 @@ handle_cast(_Msg, State) ->
 	{noreply, State}.
 
 -spec handle_info(any(), State) -> {noreply, State} when State::#state{}.
-handle_info(update, #state{universaltime=Prev, rfc1123=B1, tref=TRef}) ->
+handle_info(update, #state{universaltime=Prev, rfc1123=B1, tref=TRef0}) ->
+	erlang:cancel_timer(TRef0),
```

Comment:
No, I don't need to =)

As I've told, this pattern is just our common pattern that we are using everywhere.